packages/perseus/src/multi-items/__tests__/__snapshots__/multi-renderer_test.js.snap

```diff
@@ -162,7 +162,7 @@ exports[`multi-item renderer should snapshot: initial render 1`] = `
                                       style="border-color: transparent; border-radius: 50%;"
                                     >
                                       <div
-                                        class="libraryCircle_1tys2th"
+                                        class="circle_1tys2th"
```

Comment:
**question:** why did this class identifier change?

Reply:
When all these small components were in one file we were having to make distinctions between different parts of the style object. When I broke the components up, prefacing the style names seemed to add more noise than benefit.